syntax = "proto3";

package fbsrankings.messages.command;

import "fbsrankings/messages/options/options.proto";

message ImportSeasonByYearCommand {
    string command_id = 1;
    int32 year = 2;

    option (fbsrankings.messages.options.topic) = "fbsrankings.command.import_season_by_year";
}
